lib/polint.rb

```diff
@@ -129,36 +168,18 @@ def check_pair(key, val, contexts, is_fuzzy)
       @errors  += 1 if not_in_val.any? || not_in_key.any? || is_empty
       @fuzzies += 1 if is_fuzzy
 
-      if key =~ /(\d+):(.*)/
-        lineno = $1
-        file_and_line = "#{@pofile}:#{$1}:"
```

Comment:
So we're losing file/line references in the output logs. Not ideal, but I guess the "correct" parsing matters more.